portal-impl/src/com/liferay/portal/security/auth/AuthVerifierPipeline.java

```diff
@@ -483,7 +483,7 @@ public void modifiedService(
 					authVerifierConfiguration.getAuthVerifierClassName()));
 
 			if (_validate(authVerifierConfiguration)) {
-				_authVerifierConfigurations.add(0, authVerifierConfiguration);
```

Comment:
Dont know why it was written this way, cf627fd050cffdbf7cd654fda7b631c69934b929 this commit seems like the earliest commit for this file and it already has written like this..

Reply:
also we are removing and adding to a `CopyOnWriteArrayList`. It is not a very common operation, but it would be best if we could just validate and modify the `AuthVerifierConfiguration` in place